scripts/package.ps1

```diff
@@ -5,6 +5,10 @@ Invoke-Expression -Command "$PSScriptRoot\version.ps1"
 
 $DIR_PATH = Split-Path -Parent $MyInvocation.MyCommand.Definition
 $SRC_PATH = (Resolve-Path "$DIR_PATH\..").Path
+
+# Reference binary in ./bin/wins.exe
+Copy-Item -Force -Path $SRC_PATH\bin\wins.exe -Destination $SRC_PATH\package\windows | Out-Null
```

Comment:
Identical to stage-binary logic, but we need to do this in `ci` since this runs within the Dapper container sandbox and doesn't make it out into the host.